src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -42,18 +40,19 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
       } = loggingArgs
 
   let mutable currentModel = initialModel
+  let validationErrors = Dictionary<string, string list ref>()
```

Comment:
I don't see anywhere where this Dictionary is mutated.  Can this this continue to be read only?

Reply:
It's mutated in the member function `InitializeBinding` (line 76)

Reply:
This is restored